src/foundry/common/data/data.mjs/measuredTemplateData.d.ts

```diff
@@ -0,0 +1,117 @@
+import { FieldReturnType } from '../../../../types/helperTypes';
+import { DocumentData } from '../../abstract/module.mjs';
+import * as fields from '../fields.mjs';
+import * as documents from '../../documents.mjs';
+
+interface MeasuredTemplateDataSchema extends DocumentSchema {
+  _id: typeof fields.DOCUMENT_ID;
+  user: fields.ForeignDocumentField<{ type: documents.BaseUser; required: true }>;
+  t: {
+    type: String;
+    required: true;
+    default: typeof CONST.MEASURED_TEMPLATE_TYPES.CIRCLE;
+    validate: (t: unknown) => boolean;
+    validationError: 'Invalid {name} {field} which must be a value in CONST.MEASURED_TEMPLATE_TYPES';
+  };
+  x: typeof fields.REQUIRED_NUMBER;
+  y: typeof fields.REQUIRED_NUMBER;
+  distance: FieldReturnType<typeof fields.REQUIRED_POSITIVE_NUMBER, { default: 1 }>;
+  direction: FieldReturnType<typeof fields.ANGLE_FIELD, { default: 0 }>;
+  angle: typeof fields.ANGLE_FIELD;
+  width: FieldReturnType<typeof fields.REQUIRED_POSITIVE_NUMBER, { default: 1 }>;
+  borderColor: FieldReturnType<typeof fields.COLOR_FIELD, { required: true; default: '#000000' }>;
+  fillColor: FieldReturnType<typeof fields.COLOR_FIELD, { required: true; default: '#FF0000' }>;
+  texture: typeof fields.VIDEO_FIELD;
+  flags: typeof fields.OBJECT_FIELD;
+}
+
+interface MeasuredTemplateProperties {
+  /**
+   The _id which uniquely identifies this BaseMeasuredTemplate embedded document
+   */
+  _id: string | null;
+
+  /**
+   * The value in CONST.MEASURED_TEMPLATE_TYPES which defines the geometry type of this template
+   * @defaultValue `'circle'`
+   */
+  t: typeof CONST.MEASURED_TEMPLATE_TYPES;
```

Comment:
```suggestion
  t:  ValueOf<CONST.MeasuredTemplateTypes>;
```

Reply:
Changed it to `ValueOf<foundry.CONST.MeasuredTemplateTypes>`